main_app/models.py

```diff
@@ -46,3 +46,64 @@ class UserModel(PermissionsMixin, AbstractBaseUser):
 
     def __str__(self):
         return self.email
+
+
+class MustModel(models.Model):
+    game_id = models.IntegerField()
+    user = models.ForeignKey(UserModel, on_delete=models.CASCADE)
+    is_deleted = models.BooleanField(default=False)
+
+    def __str__(self):
+        return f"{self.user.username} {self.game_id}"
+
+
+class GameModel(models.Model):
```

Comment:
There's is no need to add `Model` as postfix. In djnago code use just Game, Genre, Screenshot, etc